pipeline/authn/authenticator_jwt_test.go

```diff
@@ -33,6 +33,8 @@ import (
 
 	"github.com/ory/x/urlx"
 
+	"github.com/ory/herodot"
+	"github.com/ory/oathkeeper/driver/configuration"
```

Comment:
```suggestion
```

Reply:
@aeneasr This is odd...my changes did not include this additional import (I only added "herodot") but github shows as if it was added. This import was there before and it is required 6 times in this module.

```
$ grep "configuration." authenticator_jwt_test.go
        "github.com/ory/oathkeeper/driver/configuration"
        viper.Set(configuration.ViperKeyAuthenticatorJWTJWKSURIs, keys)
                                        viper.Set(configuration.ViperKeyAuthenticatorJWTScopeStrategy, "exact")
                                        viper.Set(configuration.ViperKeyAuthenticatorJWTScopeStrategy, "exact")
                                        viper.Set(configuration.ViperKeyAuthenticatorJWTScopeStrategy, "none")
                                        viper.Set(configuration.ViperKeyAuthenticatorJWTScopeStrategy, "none")
```

Reply:
Yeah my bad - I accidentally messed up the merge!

Reply:
Oh, now I see, rebasing now